acceptance/tests/loader/func4x_loadable_from_modules.rb

```diff
@@ -21,6 +21,23 @@
 extend Puppet::Acceptance::TempFileUtils
 initialize_temp_dirs
 
+user = "pl#{rand(999999).to_i}"
+group = user
```

Comment:
This will fail on windows as it does not allow a local user and group to have the same name. Could prefix with `pluser` vs `plgroup`